php/helpers.php

```diff
@@ -29,9 +29,22 @@ function block_field( $name, $echo = true ) {
 		return null;
 	}
 
-	$default_fields = array( 'className' );
+	$default_fields = array( 'className' => 'string' );
 
-	if ( ! isset( $block_lab_config->fields[ $name ] ) && ! in_array( $name, $default_fields, true ) ) {
+	/**
+	 * Filters the default fields that are allowed in addition to the fields in block attributes.
+	 *
+	 * Adding an attribute to this can enable outputting it via block_field().
+	 * Normally, this function only returns or echoes Block Lab attributes (fields), and one default field.
+	 * But this allows getting block attributes that might have been added by other plugins.
+	 * To allow getting another attribute, add it to the $default_fields array.
+	 *
+	 * @param array  $default_fields An associative array of $field_name => $field_type.
```

Comment:
This could probably use more description of the types that are possible, like string, array, etc...